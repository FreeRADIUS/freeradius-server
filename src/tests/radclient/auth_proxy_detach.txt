#
#	ARGV: -i 123 -c 1 -x -F
#
User-Name = "proxy_detach",
User-Password = "hello"
