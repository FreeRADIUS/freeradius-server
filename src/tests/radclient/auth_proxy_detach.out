Sent Access-Request Id 123 from 0.0.0.0:1246 to 127.0.0.1:12351 length 45 
        User-Name = "proxy_detach"
        User-Password = "hello"
Received Access-Accept Id 123 from 127.0.0.1:12351 to 0.0.0.0:1246 via lo length 64 
        Reply-Message = "Detached subrequest sent"
(0) src/tests/radclient/auth_proxy_detach.txt response code 2
